src/ImageView.jl

```diff
@@ -440,23 +454,77 @@ function prep_contrast(img::Signal, clim::Signal{CLim{T}}) where T
         end
         rng = linspace(smin, smax, 300)
         fit(Histogram, mappedarray(nanz, vec(channelview(image))), rng; closed=:right)
+    end]
+end
+
+redview(img::AbstractMatrix{T}) where {T<:AbstractRGB} = view(channelview(img),1,:,:)
+greenview(img::AbstractMatrix{T}) where {T<:AbstractRGB} = view(channelview(img),2,:,:)
+blueview(img::AbstractMatrix{T}) where {T<:AbstractRGB} = view(channelview(img),3,:,:)
+red_clim(clim::CLim{T}) where {T<:AbstractRGB} = CLim(red(clim.min), red(clim.max))
```

Comment:
I do like this better!  I was able to remove several functions.  The only one I couldn't handle elegantly this way is the `change_red` etc family of functions in `contrast_gui.jl` because they operate on all three color channels, so I left those the same.